kinto/plugins/history/__init__.py

```diff
@@ -0,0 +1,17 @@
+from kinto.core.events import ResourceChanged
+
+from .listener import on_resource_changed
+
+
+def includeme(config):
+    config.add_api_capability('history',
+                              description='Track changes on data.',
+                              url='https://kinto.readthedocs.io')
+
+    # Activate end-points.
+    config.scan('kinto.plugins.history.views')
+
+    # Listen to every resources (except history)
```

Comment:
I wish we could select in the config the bucket that we want the history to be kept for as we did for kinto-changes.